docs/execplans/1-4-5-reply-builder.md

```
@@ -277,16 +277,18 @@ comment.
 
-    pub struct ReplyBuilder {
-        header: Option<FrameHeader>,
-        peer: SocketAddr,
-    }
-
-    impl ReplyBuilder {
-        pub fn from_frame(frame: &[u8], peer: SocketAddr) -> Self { … }
-        pub fn from_header(header: FrameHeader, peer: SocketAddr) -> Self { … }
-        pub fn error_reply(&self, error_code: u32) -> Transaction { … }
-    }
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)